charts/cadc-tap/templates/tap-deployment.yaml

```diff
@@ -30,7 +30,7 @@ spec:
           readOnly: true
         env:
         - name: CATALINA_OPTS
-          value: "-Dqservuser.jdbc.username=qsmaster -Dqservuser.jdbc.password= -Dqservuser.jdbc.driverClassName=com.mysql.cj.jdbc.Driver -Dqservuser.jdbc.url=jdbc:mysql://{{ .Values.qserv_host }}/ -Dtapuser.jdbc.username=TAP_SCHEMA -Dtapuser.jdbc.password=TAP_SCHEMA -Dtapuser.jdbc.driverClassName=com.mysql.cj.jdbc.Driver -Dtapuser.jdbc.url=jdbc:mysql://{{ template "cadc-tap.fullname" . }}-tap-schema-db:3306/ -Dca.nrc.cadc.reg.client.RegistryClient.local=true -Duws.jdbc.username=postgres -Duws.jdbc.driverClassName=org.postgresql.Driver -Duws.jdbc.url=jdbc:postgresql://{{ template "cadc-tap.fullname" . }}-uws-db/ -Dca.nrc.cadc.auth.Authenticator=org.opencadc.tap.impl.AuthenticatorImpl -Dgcs_bucket={{ .Values.gcs_bucket }} -Dgcs_bucket_url={{ .Values.gcs_bucket_url }} -Dca.nrc.cadc.util.PropertiesReader.dir=/etc/creds/"
+          value: "-Dqservuser.jdbc.username=qsmaster -Dqservuser.jdbc.password= -Dqservuser.jdbc.driverClassName=com.mysql.cj.jdbc.Driver -Dqservuser.jdbc.url=jdbc:mysql://{{ .Values.qserv_host }}/ -Dtapuser.jdbc.username=TAP_SCHEMA -Dtapuser.jdbc.password=TAP_SCHEMA -Dtapuser.jdbc.driverClassName=com.mysql.cj.jdbc.Driver -Dtapuser.jdbc.url=jdbc:mysql://{{ template "cadc-tap.fullname" . }}-tap-schema-db:3306/ -Dca.nrc.cadc.reg.client.RegistryClient.local=true -Duws.jdbc.username=postgres -Duws.jdbc.driverClassName=org.postgresql.Driver -Duws.jdbc.url=jdbc:postgresql://{{ template "cadc-tap.fullname" . }}-uws-db/ -Dca.nrc.cadc.auth.Authenticator=org.opencadc.tap.impl.AuthenticatorImpl -Dgcs_bucket={{ .Values.gcs_bucket }} -Dgcs_bucket_url={{ .Values.gcs_bucket_url }} -Dca.nrc.cadc.util.PropertiesReader.dir=/etc/creds/ -Xmx{{ .Values.jvm_max_heap_size }}"
```

Comment:
Nit: Would be nice to use `>-` here so that you can wrap this, which makes it easier to read in code review.

Reply:
Done, thanks, I think I tried this before but yaml multiline strings are not my forte. :)